src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java

```diff
@@ -725,12 +725,12 @@ private JPanel getButtonPanel() {
 		if (MainMenu.checkUserGrants("data")) {
 			buttonPanel.add(getButtonData());
 		}
-		/*
-		 * Extra / experimental / temporary features - see Admin Manual
-		 */
 		if (GeneralData.DICOMMODULEENABLED && MainMenu.checkUserGrants("btnadmdicom")) {
 			buttonPanel.add(getDICOMButton());
 		}
+		/*
+		 * Extra / experimental / temporary features - see Admin Manual
+		 */
```

Comment:
Why moving this comment?

Reply:
(it can be just removed)

Reply:
the comment was above the extra/experimental features and this PR make the DICOM button no longer experimental but part of the product so I moved the comment below the adding button code

Reply:
misunderstanding... the comment was for the method itself, like the one for getButtonLab()) (OP-845)
so it can be removed

Reply:
done